src/Application.vala

```diff
@@ -19,29 +19,12 @@
 * Authored by: Felipe Escoto <felescoto95@hotmail.com>
 */
 
-public class Wallpaperize.Application : Granite.Application {
-    public const string PROGRAM_ID = "com.github.philip-scott.wallpaperize";
+public class Wallpaperize.Application : Gtk.Application {
     public const string PROGRAM_NAME = "Wallpaperize";
```

Comment:
BTW I guess we can also remove this line since the property `program_name` that uses this constant get removed in this PR and as far as I [searched](https://github.com/Philip-Scott/wallpaperize/search?q=PROGRAM_NAME&unscoped_q=PROGRAM_NAME) it is not used in elsewhere.

Reply:
@ryonakano this constant is used in the main function as following:

```vala
Environment.set_application_name (PROGRAM_NAME);
Environment.set_prgname (PROGRAM_NAME);
```

Reply:
Ah my bad, I should have checked the whole code deeper…😅